core/invoker/src/main/scala/org/apache/openwhisk/core/invoker/Invoker.scala

```diff
@@ -91,7 +91,6 @@ object Invoker {
     Map(servicePort -> 8080.toString) ++
       ExecManifest.requiredProperties ++
       kafkaHosts ++
-      zookeeperHosts ++
```

Comment:
Without this configuration, a k8s deployment can still take advantage of the zookeeper?
I think we need to handle the zookeeper dependency in the k8s deployment first.

Reply:
Yes this just makes it such that you can start the invoker without having the config set. If you try to do that right now it will fail, but this doesn't remove the config from existing. Just that you don't need it to start.

Reply:
If you don't have it set but need it based on the arguments you passed to the invoker, it will hit this same case where the invoker errors out and stops:

`      // --uniqueName is defined with a valid value, id is empty, assign an id via zookeeper
      case CmdLineArgs(Some(unique), None, _, overwriteId) =>
        if (config.zookeeperHosts.startsWith(":") || config.zookeeperHosts.endsWith(":")) {
          abort(s"Must provide valid zookeeper host and port to use dynamicId assignment (${config.zookeeperHosts})")
        }`

Reply:
ok, I got the intention, this is just to remove the zookeeper hosts from the required configs.
It does not block adding zookeeper hosts if required.